rtf

```diff
@@ -474,16 +466,13 @@ class Platform:
           # Environment setup
           rudder_url = host.get_url()
           token = host.run('cat /var/rudder/run/api-token')
-          scenario.lib.setenv(client_path, rudder_url, token)
           for relay in relay_list:
-            # Accept the relay
-            rcli = "rudder-cli --skip-verify --url=" + rudder_url + " --token=" + token
-            command = rcli + " nodes accept "+ relay
-            (code, output) = shell(command)
-            print(output)
+            print("Accepting : " + relay)
+            # Accept the relay (double quotes must be escaped in call to run)
+            print(host.run('curl --insecure --header \\"X-API-Token: ' + token + '\\" --request POST https://localhost/rudder//api/latest/nodes/pending/' + relay + ' --data \\"status=accepted\\"'))
```

Comment:
relay should be the uuid of the node, not its hostname

Reply:
See https://docs.rudder.io/api/#operation/changePendingNodeStatus

Reply:
the relay raviale contains the uuid see the for loop above